bower.json

```diff
@@ -3,27 +3,29 @@
   "version": "0.0.1",
   "main": [],
   "dependencies": {
-    "json3": "~3.2.4",
-    "es5-shim": "~2.0.8",
     "angular": "=1.5.5",
-    "angular-bootstrap": "2.3.1",
-    "angular-breadcrumb": "=0.4.1",
-    "angular-sanitize": "=1.5.5",
+    "angular-animate": "=1.5.5",
+    "angular-i18n": "1.5.5",
     "angular-resource": "=1.5.5",
     "angular-route": "=1.5.5",
+    "angular-sanitize": "=1.5.5",
+
+    "angular-blocks": "~>0.1.8",
+    "angular-bootstrap": "2.3.1",
+    "angular-breadcrumb": "=0.4.1",
     "angular-gettext": "=1.0.0",
+    "angular-patternfly": "4.0.0-alpha.2",
```

Comment:
This isn't being used yet so I wanted to go ahead and include an alpha version so I could prototype something out with pf-table.